src/main/scala/com/exasol/cloudetl/scriptclasses/ImportScriptClassException.scala

```diff
@@ -0,0 +1,6 @@
+package com.exasol.cloudetl.scriptclasses
+
+class ImportScriptClassException(val message: String, val cause: Throwable)
```

Comment:
Not sure if you need a custom exception for this usecase, I would use some standard runtime szception